vm/stardust/vm_stardust.go

```diff
@@ -201,8 +224,8 @@ func accountStateSTVF(input *vm.ChainOutputWithCreationSlot, next *iotago.Accoun
 		return ierrors.Wrapf(iotago.ErrInvalidAccountStateTransition, "%w", err)
 	}
 
-	if err := accountBlockIssuerSTVF(input, next, vmParams); err != nil {
-		return err
+	if err := iotago.FeatureUnchanged(iotago.FeatureBlockIssuer, current.Features.MustSet(), next.Features.MustSet()); err != nil {
+		return ierrors.Wrapf(iotago.ErrInvalidAccountStateTransition, "%w", err)
```

Comment:
It would be nice to put a bit of explanation as to why the block issuer feature is not allowed to change here. For example, "block issuer feature cannot change during account state transition", or "block issuer feature can only be changed with governance transition"